packages/deployment/upgrade-test/upgrade-test-scripts/env_setup.sh

```diff
@@ -195,6 +211,9 @@ printKeys() {
   cat ~/.agoric/validator.key || true
   echo "user1: $USER1ADDR"
   cat ~/.agoric/user1.key || true
+  if [[ $THIS_NAME == "agoric-upgrade-10" || $THIS_NAME == "agoric-upgrade-11" ]]; then
```

Comment:
this is an example of why each layer should have its own env_setup. please refactor so there's a common env setup and each upgrade has its own. 

another option is to have them each inherit from env_setup of the previous, since the previous changing is going to change that image layer anyway.

Reply:
Agreed. I will refactor to include all other cases in a separate PR.